Makefile

```diff
@@ -49,5 +49,20 @@ locale:
 			msginit --no-translator -l $(LOCALE) -i $$pot -o locale/$(LOCALE)/LC_MESSAGES/`basename -s .pot $$pot`.po ; \
 		done
 
+localetest:
+	dennis-cmd lint --errorsonly locale/
+
+localeextract:
+	python manage.py extract
+	python manage.py merge
+
+localecompile:
+	cd locale; ./compile-mo.sh . ; cd --
+
+localerefresh: localeextract localetest localecompile compilejsi18n collectstatic
```

Comment:
[During the deploy](https://github.com/mozilla/kuma/blob/master/scripts/chief_deploy.py#L51-L54), ``locale/compile-mo.sh`` is called to compile the .mo files. The .mo files are [ignored](https://github.com/mozilla/kuma/blob/master/.gitignore#L31), so are not checked in.

``make localextract`` will extract any string data to ``.po`` files, which are checked in.  This needs to be done when strings are updated in code etc.

``make localerefresh`` is a new thing, to collect all the thing you need to do in development to extract strings, lint translations, and get the new translations available in your dev environment.